Cargo.toml

```diff
@@ -236,6 +236,7 @@ jsonrpc-derive = "18.0.0"
 jsonrpc-http-server = "18.0.0"
 jsonrpc-ipc-server = "18.0.0"
 jsonrpc-pubsub = "18.0.0"
+lazy-lru = "0.1.0"
```

Comment:
`lru` won't work? we already have that one

Reply:
We should replace current instances of `lru` with `lazy-lru`:
* `lru` requires `&mut self` on read-path:
https://docs.rs/lru/latest/lru/struct.LruCache.html#method.get
which can cause more lock contention because it forces a write-lock on the cache even for read.
* Also `lazy-lru` has less overhead and is generally faster.